frequency-chain/frequency-xyz
# post live updates pt. 2

# Description
- remove unused image files
- refactor assets directory
- update static images
- implement mission statement animation

- Closes #99 

## Screenshot(s)

https://github.com/user-attachments/assets/94791c25-5270-41e7-9d51-aa93d568407a

## Type of change
- Post live updates

# How to Test?
- make sure the spinner looks good in all screen sizes
- make sure static images look good
- make sure we're not missing any icons

# Checklist:
